sql-update/sick-prank.sql

```diff
@@ -0,0 +1,2 @@
+update "films"
+set "rating" = 'G';
```

Comment:
Here the should be a `where` clause so you only update movies with a rating of `NC_17`